interfaces/effect_player.hpp

```diff
@@ -82,6 +83,13 @@ namespace bnb::oep::interfaces
          */
         virtual bool call_js_method(const std::string& method, const std::string& param) = 0;
 
+        /**
+         * Evaluate the `script` in effect. This method is thread safe.
+         * @param script JS string to execute
+         * @param result_callback Callback for result, will be called in render thread.
```

Comment:
In general, in the implementation, it is not necessary to repeat docstrings because they are part of the interface. I guess we should remove them from implementation to avoid duplication and misleading descriptions if the comment was updated in the interface.